_build/test/Tests/Model/Security/modUserTest.php

```diff
@@ -105,6 +105,19 @@ public function providerGeneratePassword() {
         );
     }
 
+    public function testGeneratePasswordWithPasswordLengthOption()
+    {
+        $password = $this->user->generatePassword();
+        $this->assertNotEmpty($password);
+        $this->assertNotEquals(12, strlen($password));
+
+        $this->modx->setOption('password_generated_length',12);
```

Comment:
Also it possible to set `password_generated_length` as an empty string and it also should be checked in tests.

Reply:
I will add it to the tests! Thanks.